jlm/llvm/ir/linkage.hpp

```diff
@@ -31,6 +34,26 @@ is_externally_visible(const linkage & lnk)
   return lnk != linkage::internal_linkage;
 }
 
+static inline std::string
+linkage_to_string(const linkage & lnk)
```

Comment:
inline std::string
ToString(const linkage & lnk)

Reply:
Changed the method name